src/patternfly/components/Nav/nav.scss

```
@@ -290,17 +296,30 @@
   transition-duration: var(--#{$nav}__link--TransitionDuration--background-color), var(--#{$nav}__link--m-current--TransitionDuration--color);
   transition-property: background-color, color;
 
+  &::after {
+    position: absolute;
+    inset: 0;
+    pointer-events: none;
+    content: "";
+    border: var(--#{$nav}__link--after--BorderWidth) solid var(--#{$nav}__link--after--BorderColor);
```

Any reason to have `--after` in them? 

```suggestion
    border: var(--#{$nav}__link--BorderWidth) solid var(--#{$nav}__link--BorderColor);
```